layouts/error.vue

```diff
@@ -1,20 +1,21 @@
 <template>
   <v-app dark>
     <h1 v-if="error.statusCode === 404">
-      {{ pageNotFound }}
+      <Error404 />
     </h1>
     <h1 v-else>
       {{ otherError }}
     </h1>
-    <NuxtLink to="/">
-      Home page
-    </NuxtLink>
   </v-app>
 </template>
 
 <script>
```

Comment:
We should be using nuxt-property-decorator syntax in all components to keep consistency.